### PR TITLE
Update MapperProxy.java

### PR DESCRIPTION
**Pull request by scwlkq**
fix:代理类打印日志错误

#10486

### CODE REVIEW

File: plugin/datasource/src/main/java/com/alibaba/nacos/plugin/datasource/proxy/MapperProxy.java [hunk 1]
```diff
@@ -79,7 +79,7 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
         
         String className = mapper.getClass().getSimpleName();
         String methodName = method.getName();
-        String sql = invoke.toString();
+        String sql = ((MapperResult)invoke).getSql();
```
Comment (985492783) [inline plugin/datasource/src/main/java/com/alibaba/nacos/plugin/datasource/proxy/MapperProxy.java:82]: 有MapperResult和String两种类型，需要判断一下

**Review by 985492783 — CHANGES_REQUESTED** [review]

**Review by KomachiSion — DISMISSED** [review]

**Comment by KomachiSion** [comment]
It seems ci can't pass, please fix them.

File: plugin/datasource/src/main/java/com/alibaba/nacos/plugin/datasource/proxy/MapperProxy.java [hunk 2]
```diff
@@ -35,20 +36,20 @@
  * @author hyx
  **/
 public class MapperProxy implements InvocationHandler {
-    
+
```
Comment (KomachiSion) [inline plugin/datasource/src/main/java/com/alibaba/nacos/plugin/datasource/proxy/MapperProxy.java:39]: The indent changed, please use nacos code style to reformat code.

**Review by KomachiSion — CHANGES_REQUESTED** [review]

**Review by KomachiSion — APPROVED** [review]

**Comment by KomachiSion** [comment]
> Sorry, due to the first submission, I did not carefully read the nacos pr specification, which wasted the review time

No problem, PR keeping enhance and fix will be merged.

**Pull request merged**